dm/proto/operate.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+
+package pb;
+
+import "google/api/annotations.proto";
+
+message Operate {
+    OperateType tp = 1;
+    bytes request = 2;
+    bytes response = 3;
+    string err = 4;
+}
```

Comment:
can we support dispatching subtask to a different DM-worker instance based on `Operate` after the original one is down? or should we need to add a new mechanism?

Reply:
maybe it should rely on the heartbeat information reported by dm-worker